generators/python/src/fern_python/generators/sdk/core_utilities/client_wrapper_generator.py

```diff
@@ -612,6 +618,35 @@ def _get_bearer_auth_scheme(self) -> Optional[ir_types.BearerAuthScheme]:
             scheme_as_union = scheme.get_as_union()
             if scheme_as_union.type == "bearer":
                 return scheme_as_union
+
+        for scheme in self._context.ir.auth.schemes:
+            scheme_as_union = scheme.get_as_union()
+            if scheme_as_union.type == "oauth":
+                # TODO: For now, we create the default bearer auth scheme if the auth scheme is oauth.
+                #
+                #       This should be eventually be handled in the IR when we can support multiple auth
+                #       schemes.
+                return ir_types.BearerAuthScheme(
```

Comment:
Did we remove the "token_prefix" name from the definition? That would theoretically go here

Reply:
Great catch - I actually meant to handle this here, but it slipped my mind. It doesn't look like the `BearerAuthScheme` supports a token prefix, so this should actually be modeled as a [custom header auth](https://github.com/fern-api/fern/blob/17c5fc4b6476ee2f6500479372266e0ff451e046/packages/ir-sdk/fern/apis/ir-types-latest/definition/auth.yml#L94).

I'll drop in another TODO to focus on the use case at hand, but we can follow-up on this shortly after.